### DISCUSSION TITLE
HotReload stopped working after .net8 migration

### DISCUSSION THREAD

**Discussion by mcocaroUX** [unoplatform/uno#15435] [Q&A] [answered]
Hello, after updating to .net8 by following this DOCS:
https://platform.uno/docs/articles/migrating-from-net7-to-net8.html
https://platform.uno/docs/articles/migrating-to-uno-5.html#_content

But Im getting this error on `Windows..EnableHotReload();`
![image](https://github.com/unoplatform/uno/assets/94198647/17db63ac-04e4-4c9a-acef-15ef8c3b4c97)

[BinLogs.zip](https://github.com/unoplatform/uno/files/14239316/BinLogs.zip)

Any ideas?

**Comment by jeromelaban** [comment]
In the documentation, it is mentioned to add the namespace `using Uno.UI;`, did you also do this, so that the method `EnableHotReload` can be recognized?

**Reply by mcocaroUX** [reply]
@jeromelaban Yes, I added `using Uno.UI;` It does not recognize it only when targeting windows, but build fails on other platforms too. If I remove `Windows.EnableHotReload();` it app builds correctly on all platforms

**Reply by jeromelaban** [reply, marked as answer]
Thanks. You can make a comparison with a template built using the new project wizard, and see what may be missing. That particular line is available in the templates. If the `using Uno.UI` is present at the top of the file, you should not be getting this error.

**Reply by mcocaroUX** [reply]
@jeromelaban Thanks for your help, so the only difference I found was on the `.csproj` we were still using `Microsoft.NET.Sdk` (Project is using a template from 8 months ago) I migrated to `Uno.Sdk` and that solved it

**Reply by jeromelaban** [reply]
Thanks for the update!
